server_admin/topics/authentication/conditions.adoc

```diff
@@ -0,0 +1,43 @@
+=== Conditions in Conditional flows
+
+As was mentioned in <<_execution-requirements, Execution requirements>>, _Condition_ executions can be only contained in _Conditional_ subflow.
+If all _Condition_ executions evaluate as true, then the _Conditional_ sub-flow acts as _Required_.
+You can process the next execution in the _Conditional_ sub-flow.
+If some executions included in the _Conditional_ sub-flow evaluate as false, then the whole sub-flow is considered as _Disabled_.
+
+==== Available Conditions
+
+`Condition - User Role`::
+This execution has the ability to determine if the user has role defined by _User role_ field.
+If the user has the required role, the execution is considered as true and other executions are evaluated.
+The administrator has to define following fields:
+
+Alias:::
+Describes a name of the execution, which will be shown in the authentication flow.
+
+User role:::
+Role the user should have to execute this flow.
+To specify an application role the syntax is `appname.approle`, i.e. `myapp.myrole`.
+
+`Condition - User Configured`::
+This checks if the other executions in the flow are configured for the user.
+There is an example in Execution requirements section with the OTP form.
```

Comment:
Try to avoid "there is" or "it is" -- It is common but makes no sense grammatically.  What is the subject of the sentence? "There"
Rewording also makes it more concise.
How about "The Execution requirements section includes an example of the OTP form."